#1069 - RefactorUncertaintySamplingStrategy

chaitraliagashe opened the pull request:
**What's in the PR**
* Refactor UncertaintySamplingStrategy.java.
* Removed code not related to Sampling Strategy from UncertaintySamplingStrategy.java
* Added this code to ActiveLearningSidebar.java
  * The generateNextSuggestion() method in this class contains a lot of code that is not directly relevant to generating the next set of candidate suggestions based on AL strategy. This code should be moved to ActiveLearningSidebar.java from where generateNextSuggestion() method is called.

**How to test manually**
* Start inception server and run the AL mode. There should be no change in the user experience.

**Automatic testing**
* [ ] PR does not include Automated Tests

**Documentation**
* [ ] PR does not update documentation

ukp-svc-jenkins commented:
Can one of the admins verify this patch?

reckart commented:
Jenkins, can you test this please?

ukp-svc-jenkins commented:
[![49% (0.0%) vs master 49%](https://zoidberg.ukp.informatik.tu-darmstadt.de/jenkins//coverage-status-icon/?coverage=0.4931&masterCoverage=0.4931)](https://zoidberg.ukp.informatik.tu-darmstadt.de:443/jenkins/job/INCEpTION%20(GitHub)%20(PR%20builder)/2861/)

reckart commented:
@chaitraliagashe I have fixed the commit message from your commit and moved the code to fetch the next suggestion from the AL sidebar into the AL service to avoid having too much logic accumulating in the UI layer.

reckart commented:
Jenkins, can you test this please?

ukp-svc-jenkins commented:
[![49% (0.0%) vs master 49%](https://zoidberg.ukp.informatik.tu-darmstadt.de/jenkins//coverage-status-icon/?coverage=0.4927&masterCoverage=0.4927)](https://zoidberg.ukp.informatik.tu-darmstadt.de:443/jenkins/job/INCEpTION%20(GitHub)%20(PR%20builder)/2878/)

chaitraliagashe commented:
Ohh.. makes sense.. thanks for correcting..

On Fri, May 3, 2019, 3:36 PM Richard Eckart de Castilho <
notifications@github.com> wrote:

> @chaitraliagashe <https://github.com/chaitraliagashe> I have fixed the
> commit message from your commit and moved the code to fetch the next
> suggestion from the AL sidebar into the AL service to avoid having too much
> logic accumulating in the UI layer.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/inception-project/inception/pull/1141#issuecomment-489043908>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALFXVSGS2Q2VEK3XZUUKOGDPTQFCJANCNFSM4HJBN3OA>
> .
>

chaitraliagashe commented:
Hi Richard,
Please let me know after you have merged the changes. I will start my work
on test-case then.

Regards,
Chaitrali

On Sat, May 4, 2019 at 8:16 AM chaitrali agashe <chaitraliagashe@gmail.com>
wrote:

> Ohh.. makes sense.. thanks for correcting..
>
> On Fri, May 3, 2019, 3:36 PM Richard Eckart de Castilho <
> notifications@github.com> wrote:
>
>> @chaitraliagashe <https://github.com/chaitraliagashe> I have fixed the
>> commit message from your commit and moved the code to fetch the next
>> suggestion from the AL sidebar into the AL service to avoid having too much
>> logic accumulating in the UI layer.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/inception-project/inception/pull/1141#issuecomment-489043908>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ALFXVSGS2Q2VEK3XZUUKOGDPTQFCJANCNFSM4HJBN3OA>
>> .
>>
>

-- 
~Chaitrali

reckart commented:
Jenkins, can you test this please?

ukp-svc-jenkins commented:
[![49% (0.0%) vs master 49%](https://zoidberg.ukp.informatik.tu-darmstadt.de/jenkins//coverage-status-icon/?coverage=0.4922&masterCoverage=0.4922)](https://zoidberg.ukp.informatik.tu-darmstadt.de:443/jenkins/job/INCEpTION%20(GitHub)%20(PR%20builder)/2898/)

reckart commented:
@UWinch could you briefly test if the active learning still works ok and approve this PR if this is the case?

chaitraliagashe commented:
Thanks @UWinch, Richard.

On Tue, May 7, 2019 at 8:16 PM UWinch <notifications@github.com> wrote:

> Merged #1141 <https://github.com/inception-project/inception/pull/1141>
> into master.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/inception-project/inception/pull/1141#event-2324461037>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALFXVSBLBITP3ASN4N2F4ZDPUGI4FANCNFSM4HJBN3OA>
> .
>


-- 
~Chaitrali
